src/test/java/org/elasticsearch/action/search/simple-msearch1.json

```diff
@@ -4,7 +4,7 @@
 {"query" : {"match_all" {}}}
 {}
 {"query" : {"match_all" {}}}
-{"search_type" : "count"}
+{"search_type" : "dfs_query_then_fetch"}
```

Comment:
This was only used by a unit test to check that msearch supports overriding the search_type per request, so I think it's fine (the test only parses the request, it does not run it).